meta-lmp-base/recipes-support/lmp-auto-hostname/lmp-auto-hostname/lmp-auto-hostname.service.in

```diff
@@ -5,8 +5,9 @@ Before=network-pre.target avahi-daemon.service
 After=local-fs.target dbus.service
 
 [Service]
-Environment=MACHINE=@@LMP_HOSTNAME_MACHINE@@ MODE=@@LMP_HOSTNAME_MODE@@ NETDEVICE=@@LMP_HOSTNAME_NETDEVICE@@
+Environment=MACHINE=@@LMP_HOSTNAME_MACHINE@@ MODE=@@LMP_HOSTNAME_MODE@@ NETDEVICE=@@LMP_HOSTNAME_NETDEVICE@@ FIOVB_VAR=@@LMP_HOSTNAME_FIOVB_VAR@@
 ExecStart=/usr/bin/lmp-update-hostname
+ExecStartPost=/bin/systemctl disable lmp-auto-hostname.service
```

Comment:
Because we use `usrmerge` in lmp the systemctl binary are placed in `/usr/bin/systemctl`

Reply:
Good remark, I have amended my commit.

Reply:
Thanks for your contribution.